test/UnitTest/Microsoft.AspNet.OData.Test.Shared/Builder/EnumTypeTest.cs

```diff
@@ -1192,10 +1201,16 @@ public enum Life
     {
         [EnumMember(Value = "happy")]
         Happy = 1,
+
         [EnumMember(Value = "sad")]
         Sad = 2,
+
         JustSoSo = 3,
+
         [EnumMember]
-        KeepDefaultName
+        KeepDefaultName = 4,
+
+        [EnumMember(Value = "soso")]
```

Comment:
Is the mismatched value part of the test case?

Reply:
That's intentionally. I want to test the model alias.